notes/20200217_meeting.md

```diff
@@ -34,6 +33,7 @@ Other benefits of RAP include:
 ### Improved quality: 
 - no manual steps (in the part of the process selected for RAP)
 - peer review of code
+- code modularity (i.e. functions / less code repetition)
```

Comment:
Similar to other comments I've made, do people think this is required for MVP RAP?

Reply:
It depends how much we want the MVP to represent improved quality :)

Reply:
I think we definitely do, it's one of the most important benefits! Removing manual steps and implementing peer review will improve quality. I guess it's a scale, so it depends on our threshold for quality and how much of an improvement having code in functions would bring versus the barrier it presents for people who are new to programming.

Reply:
I think I raised this question briefly on Slack, but I'll ask it again here. Should peer review be part of RAP or should RAP enable peer review? If we separate peer review from RAP we empower analysts to create a RAP themselves. There's a certain elegance to separating state (reproducibility) from process (peer review).

Instead of peer review, which reviews all aspects of quality, we could instead focus on peer 'verification' (for lack of a better word), where the sole purpose is to verify that the analysis can be reproduced on someone else's system with minimal guidance.

Reply:
Really interesting points, thanks @jl5000 - let's discuss on Thursday. It seems to me that there are two types of 'review' - one is ensuring the quality of the work and the other is ensuring that it can be easily replicated and developed. I think that for RAP both are essential - a RAP should be high quality but it also shouldn't be a black box created by one person.

Reply:
I agree with Anna! I think there has to be some element of peer review within the team/department to make sure the process is robust and sustainable. When we work with a team on RAP we ask that at least 2 people be involved in the process so that there should always be someone who understands the process. We also review the code separately and look at where improvements could be made or where things could be done more efficiently.

Reply:
We're quite resource limited, so even having an additional one person involved in every process would be a huge strain in what we can deliver. Would a RAP with good documentation and reproducibility practices offset the need for additional analysts to understand the process?

Reply:
I don't think that peer review necessarily needs to be hugely burdensome - it could be someone doing a one-off review or it could be more than one person working on a project. But I think for both the quality and sustainability of the work it's hugely important. I don't think reproducibility is the only requirement of a RAP - one of the reasons I wish it had had a different name!

Reply:
Ah, terminology may be the issue then. Do you see RAP more as a general mark of quality?

Reply:
I think the quality of the documentation can only be proved by peer review. 

Reproducibility is about giving future users/maintainers of the analysis the ability to run the analysis again. This can only be achieved through adequate documentation. The only assurance that adequate documentation has been provided is to test that with someone outside the project team.